samples/java_springboot/19.custom-dialogs/src/main/java/com/microsoft/bot/sample/customdialogs/DialogBot.java

```diff
@@ -48,6 +50,10 @@ public DialogBot(
     protected CompletableFuture<Void> onMessageActivity(
         TurnContext turnContext
     ) {
+        Logger logger = Logger.getLogger(DialogBot.class.getName());
+        logger.info("Running dialog with Message Activity.");
```

Comment:
Usually, we are doing this as follows:

```suggestion
        Logger.getLogger(DialogBot.class).info("Running dialog with Message Activity.");
```

as the object logger won't be used anymore here, not necessary to create an object for this